types/validator/index.d.ts

```diff
@@ -13,1237 +13,1237 @@
-     * Please note that when `all_lowercase` is `true`, iCloud addresses are lowercased regardless of the value of this setting.
+     * Remove characters that do not appear in the whitelist.
      *
-     * @default true
+     * @param chars - The characters are used in a `RegExp` and so you will need to escape some chars, e.g. `whitelist(input, '\\[\\]')`.
      */
-    icloud_lowercase?: boolean;
+    function whitelist(input: string, chars: string): string;
+
     /**
-     * Normalizes addresses by removing "sub-addresses", which is the part following a `"+"` sign
-     * (e.g. `"foo+bar@icloud.com"` becomes `"foo@icloud.com"`).
-     *
-     * @default true
+     * Converts to string.
      */
-    icloud_remove_subaddress?: boolean;
+    function toString(input: any): string;
 }
 
-/**
- * Canonicalizes an email address. (This doesn't validate that the input is an email, if you want to validate the email use `isEmail` beforehand)
- *
- * @param [options] - Options
- */
-export function normalizeEmail(email: string, options?: NormalizeEmailOptions): string | false;
-
-/**
- * Trim characters from the right-side of the input.
- *
- * @param [chars] - characters (defaults to whitespace)
- */
-export function rtrim(input: string, chars?: string): string;
-
-/**
- * Remove characters with a numerical value < `32` and `127`, mostly control characters.
- * Unicode-safe in JavaScript.
- *
- * @param [keep_new_lines=false] - if `true`, newline characters are preserved (`\n` and `\r`, hex `0xA` and `0xD`).
- */
-export function stripLow(input: string, keep_new_lines?: boolean): string;
-
-/**
- * Convert the input string to a boolean.
- * Everything except for `'0'`, `'false'` and `''` returns `true`.
- *
- * @param [strict=false] - in `strict` mode, only `'1'` and `'true'` return `true`.
- */
-export function toBoolean(input: string, strict?: boolean): boolean;
-
-/**
- * Convert the input string to a `Date`, or `null` if the input is not a date.
- */
-export function toDate(input: string): Date | null;
-
-/**
- * Convert the input string to a float, or `NaN` if the input is not a float.
- */
-export function toFloat(input: string): number;
-
-/**
- * Convert the input string to an integer, or `NaN` if the input is not an integer.
- *
- * @param [radix=10] - radix or base (defaults to 10)
- */
-export function toInt(input: string, radix?: number): number;
-
-/**
- * Trim characters from both sides of the input.
- *
- * @param [chars] - characters (defaults to whitespace)
- */
-export function trim(input: string, chars?: string): string;
-
-/**
- * Remove characters that do not appear in the whitelist.
- *
- * @param chars - The characters are used in a `RegExp` and so you will need to escape some chars, e.g. `whitelist(input, '\\[\\]')`.
- */
-export function whitelist(input: string, chars: string): string;
-
-/**
- * Converts to string.
- */
-export function toString(input: any): string;
-
-import * as Validator from './';
```

Comment:
this is the line that was causing the bug

Reply:
Was just coming to fix this. Thanks!